packages/cycle-scripts-one-fits-all/template/tsconfig.json

```diff
@@ -2,7 +2,7 @@
     "compileOnSave": false,
     "compilerOptions":
     {
-        "module": "ES6",
```

Comment:
YEs, I was just looking at tha tand agree should follow the docs! will revert. Not I think the ES5 is redundant, Did you include it for a reason?